dev-python/spyder-kernels/spyder-kernels-1.9.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Is it possible to add Python 3.8 support?

Reply:
Same, as pyls. To be more specific I need py3_8 support on jupyter and friends maintained by the science team.

It would be great to add py3_8 to all these ebuilds, especially considering spyder officially supports py3_8 from this release onwards. But, I'm not going to make a huge PR that adds py3_8 to all the dependencies that need it. Such a large PR will just get ignored, or it will take very long to be reviewed, and by that time the science team will likely have added py3_8 to their ebuilds anyway.

Tl;dr Yes it is possible but it will make the PR huge, and therefore will likely not be worth the effort.